test/unit/gateways/omise_test.rb

```diff
@@ -0,0 +1,646 @@
+require 'test_helper'
+
+class OmiseTest < Test::Unit::TestCase
+  def setup
+    @gateway = OmiseGateway.new(
+      public_key: 'pkey_test_abc',
+      secret_key: 'skey_test_123',
+    )
+
+    @credit_card = credit_card
+    @amount = 3333
+
+    @options = {
+      order_id: '1',
+      billing_address: address,
+      description: 'Store Purchase'
+    }
+
+    @card_token = {
+      object: 'token',
+      id: 'tokn_test_4zgf1crg50rdb68xlk5'
+    };
+  end
+
+  def test_supported_countries
+    assert @gateway.supported_countries == %w( TH )
+  end
+
+  def test_supports_scrubbing
+    assert @gateway.supports_scrubbing? == true
+  end
+
+  def test_scrub
+    assert @gateway.scrub(
```

Comment:
This test doesn't do any checks for card number or CVC scrubbing - can you add them?